.github/workflows/bff-ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [macOS-latest, ubuntu-latest, windows-latest]
+        runs-on: [ubuntu-latest, windows-latest]
```

Comment:
Removing mac-os from the matrix. In a later PR, we'll only build on linux and do code signing there